clang/test/CodeGenCXX/modules-vtable.cppm

```diff
@@ -41,9 +43,10 @@ Base::~Base() {}
 // CHECK: @_ZTSW3Mod4Base = constant
 // CHECK: @_ZTIW3Mod4Base = constant
 
-// CHECK-INLINE: @_ZTVW3Mod4Base = linkonce_odr {{.*}}unnamed_addr constant
-// CHECK-INLINE: @_ZTSW3Mod4Base = linkonce_odr {{.*}}constant
-// CHECK-INLINE: @_ZTIW3Mod4Base = linkonce_odr {{.*}}constant
+// With the new Itanium C++ ABI, the linkage of vtables in modules don't need to be linkonce ODR.
```

Comment:
Would they benefit from still being linkonce? What if the vtable is never used? Or do we leave that up to `--gc-sections` linker behavior? (I'm not clear on when we use linkage to let things be dropped, and when we use the linker function-sections/data-sections/gc-sections behavior to drop unused stuff)

Reply:
If it is linkonce, as long as I understand correctly, if the vtable is not used in the current TU, the middle end is free to remove the unused linkonce entity. But this is not wanted. Since all the things in the module purview is potentially used. Do I misunderstand anything?

Reply:
right, sorry, was misreading the linkages - I guess `weak` might be applicable? 
> “weak” linkage has the same merging semantics as linkonce linkage, except that unreferenced globals with weak linkage may not be discarded. 
But maybe just normal external linkage is the right thing & we let the linker GC things if the user asks it to. Fair enough.

Reply:
While I agree `weak` may work here, I feel it is odd. I feel the default strong linkage seems better when it is applicable. What's the advance to use `weak` linkage here?